admin/development.ini

```diff
@@ -6,58 +6,59 @@
 [app:main]
 use = egg:c2cgeoportal_admin
 
-pyramid.reload_templates = true
-pyramid.debug_authorization = false
-pyramid.debug_notfound = false
-pyramid.debug_routematch = false
+pyramid.reload_templates = %(DEVELOPMENT)s
+pyramid.debug_authorization = %(DEVELOPMENT)s
+pyramid.debug_notfound = %(DEVELOPMENT)s
+pyramid.debug_routematch = %(DEVELOPMENT)s
 pyramid.default_locale_name = en
 pyramid.available_languages = fr en
 pyramid.includes =
-    pyramid_debugtoolbar
 
-sqlalchemy.url = postgresql://www-data:www-data@localhost/c2cgeoportal
+sqlalchemy.url = %(SQLALCHEMY_URL)s
+sqlalchemy.pool_recycle = 30
+sqlalchemy.pool_size = %(SQLALCHEMY_POOL_SIZE)s
+sqlalchemy.max_overflow = %(SQLALCHEMY_MAX_OVERFLOW)s
+
+sqlalchemy_slave.url = %(SQLALCHEMY_URL_SLAVE)s
+sqlalchemy_slave.pool_recycle = 30
+sqlalchemy_slave.pool_size = %(SQLALCHEMY_POOL_SIZE)s
+sqlalchemy_slave.max_overflow = %(SQLALCHEMY_MAX_OVERFLOW)s
+
 schema = main
 srid = 3857
 
 retry.attempts = 3
 
-# By default, the toolbar only appears for clients from IP addresses
-# '127.0.0.1' and '::1'.
-# debugtoolbar.hosts = 127.0.0.1 ::1
-
-###
-# wsgi server configuration
-###
-
-[server:main]
```

Comment:
After removing [server:main] section, make serve`` should fail, is this stricly needed